crates/client/gateway/src/client/request_builder.rs

```diff
@@ -71,44 +75,74 @@ impl<'a> RequestBuilder<'a> {
         unpack(self.send_get_raw().await?).await
     }
 
-    pub async fn send_get_raw(self) -> Result<Response, SequencerError> {
-        self.client.get(self.url).headers(self.headers).query(&self.params).send().await.map_err(Into::into)
+    pub async fn send_get_raw(self) -> Result<Response<Incoming>, SequencerError> {
+        let uri = self.build_uri()?;
+
+        let mut req_builder = Request::builder().method("GET").uri(uri);
```

Comment:
Consider using `http::Method::GET` instead.
```suggestion
        let mut req_builder = Request::builder().method(http::Method::GET.as_str()).uri(uri);
```